data/tls-listener.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @title TCP/TLS Listener: Ping
+ * @difficulty intermediate
+ * @tags cli
+ * @run --allow-net --allow-read <url>
+ * @resource {https://deno.land/api?s=Deno.listenTls} Doc: Deno.listenTls
+ *
+ * An example of a TCP listener using TLS on localhost that will log the message if written to and close the connection if connected to.
+ */
+
+// Instantiate an instance of a TCP listener on localhost port 443.
+const listener = Deno.listenTls({
+  hostname: "127.0.0.1",
+  port: 443,
+  transport: "tcp",
+  certFile: "./server.crt",
+  keyFile: "./server.key",
```

Comment:
Let's use `cert` and `key` options instead as these look deprecated now https://deno.land/api@v1.36.0?s=Deno.ListenTlsOptions